ORBIT/phases/design/oss_design_floating.py

```diff
@@ -151,18 +150,19 @@ def calc_num_mpt_and_rating(self):
         capacity = num_turbines * turbine_rating
 
         self.num_substations = _design.get(
-            "num_substations", int(np.ceil(capacity / 500))
+            "num_substations",
+            int(np.ceil(capacity / 500)),
         )
         self.num_mpt = np.ceil(
-            num_turbines * turbine_rating / (250 * self.num_substations)
+            num_turbines * turbine_rating / (250 * self.num_substations),
```

Comment:
Ya, I might need to turn that setting off. Let me give the diff another look to see how widespread that pattern is because I'll admit that it's a bit weird.